views/admin_edit_rhapsody.handlebars

```diff
@@ -0,0 +1,147 @@
+                                    <div class="col-md-9">
+                                        <input type="text" name="further_scripture" value="{{viewData.rhapsody.further_scripture}}"
+                                            class="form-control">
+                                        <small class="form-control-feedback"> Enter the further scripture of the
+                                            rhapsody</small> </div>
+                                </div>
+                                <div class="form-group row">
+                                    <label class="control-label text-right col-md-3">One Year Bible Plan</label>
+                                    <div class="col-md-9">
+                                        <input type="text" name="one_year_bible_plan" value="{{viewData.rhapsody.one_year_bible_plan}}"
+                                            class="form-control">
+                                        <small class="form-control-feedback"> Enter the one year bible plan of the
+                                            rhapsody</small> </div>
+                                </div>
+                                <div class="form-group row">
+                                    <label class="control-label text-right col-md-3">Date</label>
+                                    <div class="col-md-9">
+                                        <input id="rhapsodyDate" type="date" name="date" class="form-control">
+                                        <small class="form-control-feedback"> Enter the date of the rhapsody</small>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="form-actions">
+                                <div class="row">
+                                    <div class="col-md-12" style="padding:20px">
+                                        <div class="row">
+                                            <div class="offset-sm-3 col-md-9">
+                                                <button type="submit" name="save&stay" class="btn btn-success"> <i
+                                                        class="fa fa-check"></i>Save & Stay</button>
+                                                <button type="button" name="save&exit" class="btn btn-success">Save
+                                                    & Exit</button>
+                                                <button type="button" name="cancel" class="btn btn-info">Cancel</button>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Row -->
+    </div>
+</div>
```

Comment:
EOF
end of file , need new line. i think there is a setting you can make on your IDE to add a new line automatically after save